Assets/Mirror/Editor/Weaver/Extensions.cs

```diff
@@ -84,6 +84,16 @@ public static bool IsMultidimensionalArray(this TypeReference tr)
             return tr is ArrayType arrayType && arrayType.Rank > 1;
         }
 
+        /// <summary>
+        /// Is the field a NetworkIdentity or GameObject
+        /// </summary>
+        public static bool IsNetworkIdentityField(this FieldDefinition fd)
```

Comment:
I would make this receive the TypeReference instead.
Then this becomes reusable for parameters too.